frontend/src/components/FifthExamComponent/FifthExamFormComponent.tsx

```diff
@@ -0,0 +1,164 @@
+  const aDate = new Date();
+  const dueDate = `${aDate.getDate()}.${aDate.getMonth() + 1}.${aDate.getFullYear()}`;
+
+  const [examType, setExamType] = useState('');
+  const [studentName, setStudentName] = useState('');
+  const [studentPartner, setStudentPartner] = useState('');
+  const [chosenTopicArea, setChosenTopicArea] = useState('');
+  const [referenzfach, setReferenzfach] = useState('');
+  const [examiner, setExaminer] = useState('');
+  const [bezugsfach, setBezugsfach] = useState('');
+
+  const referenzfachList = referenzfachMock;
+  const examinerList = examinerMock;
+  const bezugsfachList = bezugsfachMock;
+
+  const handleExamTypeInputChange = (event) => {
+    setExamType(event.target.value);
+  };
+  const handleStudentNameInputChange = (event) => {
+    setStudentName(event.target.value);
+  };
+  const handleStudentPartnerInputChange = (event) => {
+    setStudentPartner(event.target.value);
+  };
+  const handleChosenTopicAreaInputChange = (event) => {
+    setChosenTopicArea(event.target.value);
+  };
+  const handleReferenzfachInputChange = (event) => {
+    setReferenzfach(event.target.value);
+  };
+  const handleExaminerInputChange = (event) => {
+    setExaminer(event.target.value);
+  };
+  const handleBezugsfachInputChange = (event) => {
+    setBezugsfach(event.target.value);
+  };
+
+  const handleGeneratePDF = () => {
+    alert('pdf generiert...');
+    const pdf = {
+      examType,
+      studentName,
+      studentPartner,
+      chosenTopicArea,
+      referenzfach,
+      examiner,
+      bezugsfach,
+    };
+    console.log(pdf.toString());
+  };
+
+  const handleSubmitFifthExamForm = () => {};
+
+  return (
+    <div style={{ overflow: 'scroll' }}>
+      <Paper
+        className="fifthExamPaper"
+        style={{
+          display: 'flex', flexDirection: 'column', width: '70%', marginRight: '3vw', padding: '5vw', margin: 'auto',
+        }}
+      >
+        <h1 id="headingFifthExaam">Pruefungskomponente</h1>
+
+        <FormControl component="fieldset">
+          <FormLabel component="legend" />
+          <RadioGroup
+            aria-label="Präsentationsprüfung"
+            name="radio-buttons-group"
+            row
+          >
+            <FormControlLabel
+              value="Präsentationsprüfung"
+              control={<Radio />}
+              label="Präsentationsprüfung"
+              onChange={handleExamTypeInputChange}
+            />
+            <FormControlLabel value="Besondere Lernleistung" control={<Radio />} label="Besondere Lernleistung" />
+          </RadioGroup>
+        </FormControl>
+
+        <TextField label="Prüfling" variant="outlined" style={{ marginTop: '2vh' }} onChange={handleStudentNameInputChange} />
+
+        <TextField label="ggf. Partner:in" variant="outlined" style={{ marginTop: '2vh' }} onChange={handleStudentPartnerInputChange} />
+
+        <FormControl style={{ width: '100%', marginTop: '2vh' }} variant="outlined">
```

Comment:
kleiner Tipp: wenn du wie in diesem Form immer als marginTop: '2vh' hast, kannst (bzw. solltest) du die variable auslagern. weil wenn du mal den abstand zwischen den elementen verändern willst, willst du das ja nur an einer stelle ändern müssen. Dafür machst du folgendes:
Du legst dir ein objekt als constante an:
  const topDistanceToPreviousInputField = {marginTop: "2vh"}
dann bei der styleprop von deinen komponenten (z.B. FormControl) kannst du entweder einfach als machen: style={topDistanceToPreviousInputField } dann wird dort dein object als styleobject genommen.

Willst du aber noch zusätzlich inline styles reinpacken (wie z.B. width: "100%") wirds entwas schwieriger.

style= kann nur ein object bekommen, das heißt sowas wie style={{width: "100%"},{styleobject}} geht nicht. Die beiden objekte musst du vorher mergen. das geht so:

Object.assign({}, {firstObject}, {secondObject}) also z.B. Object.assign({},{width: '20%'}, distance)
wenn firstObject und secondObject den gleichen wert haben, dann kann natürlich nur einer dieser werte genommen werden. Dann überschreibt der wert von secondObject den von firstObject. Das kann wichtig sein, wenn du z.B. ein
defaultInputStyleObject = {marginTop: "2vh", width: "100%"} hast und die margin zwar übernehmen willst, nicht aber die width. dann kannst du als secondObject ein {width: "50%} übergeben und überschreibst dadurch den wert vom defaultInputStyleObject.

Dass nur so als kleiner exkurs. ich überlass es dir ob du es hier so wie vorgeschlagen machst oder so belässt. am ende sinds halt nur ne handvoll änderungen. Aber ich wollte mein wissen mal trotzdem weitergeben :D

Reply:
ja stimmt habe ich mich auch schon gefragt - sieht professioneller aus als meine Idee aber rein vom logischen her könnte man auch einfach 2vw als globale Variable speichern oder?